src/analytics/selectors.ts

```diff
@@ -24,7 +24,7 @@ export const getCurrentUserTraits = createSelector(
     tokensWithTokenBalanceSelector,
     coreTokensSelector,
     getLocalCurrencyCode,
-    numberVerifiedSelector,
+    phoneVerificationStatusSelector,
```

Comment:
open to suggestions for how else to do this, it seems like `createSelector` expects to have a maximum of 12 selectors as input. adding a 13th one makes typescript complain :/ i found an [old issue](https://github.com/reduxjs/reselect/issues/378) that references the problem, and the suggestion was to use a `createSelector` as an input...